tests/integration/nerdctl/gha-run.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+#
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+kata_tarball_dir="${2:-kata-artifacts}"
+nerdctl_dir="$(dirname "$(readlink -f "$0")")" 
+source "${nerdctl_dir}/../../common.bash"
+
+function install_dependencies() {
+	# Install pip as we're going to install `lastversion` from it
+	sudo apt update
+	sudo apt -y install pip
+
+	# Install lastversion from pip
+	#
+	# --break-system-packages is, unfortunately, needed here as it'll
+	# also bring in some python3 dependencies on its own
+	pip install lastversion --break-system-packages
```

Comment:
When testing this locally on Ubuntu 20.04 I hit:
```
# pip install lastversion --break-system-packages

Usage:
  pip install [options] <requirement specifier> [package-index-options] ...
  pip install [options] -r <requirements file> [package-index-options] ...
  pip install [options] [-e] <vcs project url> ...
  pip install [options] [-e] <local project path> ...
  pip install [options] <archive url/path> ...
```
I'm guessing you tested it on 23.04 and it's ok there?

Reply:
I tested this with 23.04, but I just found a few steps that I missed, let me update the PR here, and also the steps, and paste a full log of what I'm doing.

Reply:
@stevenhorsman, here's the full history of what I've done: https://gist.github.com/fidencio/abfbfe72f8e6b2680b80a8376c8e556e

Reply:
Thanks for the evidence that it works for you!